src/main/scala/no/ndla/learningpathapi/service/search/SearchServiceComponent.scala

```diff
@@ -65,6 +128,32 @@ trait SearchServiceComponent extends LazyLogging {
         pageSize)
     }
 
+    def allV2(withIdIn: List[Long], taggedWith: Option[String], sort: Sort.Value, language: Option[String], page: Option[Int], pageSize: Option[Int]): SearchResult = {
+
+      val fullQuery = language match {
+        case None => QueryBuilders.boolQuery()
+        case Some(_) => {
```

Comment:
Om du gjør om denne til `Some(lang)` kan du bruke `lang` under slik `...s"titles.$lang"...` i stedet for `..s".${language.getOrElse("")...`